content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -23,11 +23,11 @@ tags:
 title: Financial Services Use in Tanzania
 ---
 
-## Background
+## About DataSet
```

Comment:
How about "About the dataset"? We always need our writing to look professional because otherwise some of our learners start to doubt the quality of the content.